INSTALL.txt

```diff
@@ -273,7 +273,29 @@ aliases to provide the pd command.
 
 ### Windows
 
-*Windows-specific info goes here*
+Pd on Windows is built with either MinGW or Cygwin.
+
+available audio APIs which can be enabled/disabled via ./configure are:
+* MMIO (slow!) 	--enable-mmio --disable-mmio
+* ASIO			--enable-asio --disable-asio
+* JACK			--enable-jack --disable-jack
+
+Note: because of license restrictions, Pd can't ship the necessary ASIO files. 
+If you want to use ASIO, have a look at /asio/README.txt for further instructions.
+
+The standard 'make install' won't work outside your Cygwin/Msys2 environment (if at all).
```

Comment:
Can we use double quote around "make install" etc to match the rest of the document?